easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-2.3.1-intel-2015a-eb-deps.eb

```diff
@@ -0,0 +1,51 @@
+name = 'OpenFOAM'
+version = '2.3.1'
+versionsuffix = '-eb-deps'
```

Comment:
hmm, maybe just `-deps`? or just `-eb`?

not sure, but can't find a better one either myself